client/src/App.css

```diff
@@ -1,5 +1,7 @@
 .App {
-  text-align: center;
+  justify-content: center;
+  background-color: whitesmoke;
+  display: flex;
```

Comment:
We should be keeping styles inside of makeStyles function. This file is from the boilerplate and can probably be removed

Reply:
I moved the css into a makeStyles function for the components